### DISCUSSION TITLE
Autoplay consecutive videos as muted/unmuted

### DISCUSSION THREAD

**Discussion by MkManish25** [videojs/video.js#8659] [Q&A]
### Description

I am using videojs in my project and developed funtionality similar to youtube shorts. The challenge I am facing is not able to play consecutive videos after every swipe as autoplay with sound, after every swipe i have to tap on unmute first and then afterwards only i can hear sound. I want to maintain a state so that once a video is unmuted all the consecutive videos should be played unmuted till the time user has clicked on mute button again. Facing this issue in all the major browsers in IOS like chrome, safari,etc.

### Reduced test case

https://m.youtube.com/shorts

### Steps to reproduce

1.Opening any short and once tapped on mute button all the next videos in current session should be played unmuted
2. All the videos should be played with sound till the time user has again tap on mute button
4. how to make resemble video play with audio in videojs

### What version of Video.js are you using?

3.0.2

### What browser(s) including version(s) does this occur with?

All browsers 

### What OS(es) and version(s) does this occur with?

IOS

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! 👋

If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
To help make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/videojs/video.js/blob/master/CONTRIBUTING.md).

**Comment by BrainCrumbz** [comment]
Hi there. Two sets of answers, first on the topic, next on the how this issue is described.

*On the topic:*

One way to obtain what you are after is to prepare two video players, one visible, one hidden, when you web page is loaded. And never destroy nor remove those video players.  
On the visible player you set the source of your current clip, on the hidden player you set the source of your next clip. When the visible player ends playback, your code:
* shows the hidden player and makes it play 
* hides the visible player and prepares it for a further clip

Given that the user interacted with page upon the first time it clicks to play the first clip, from then on the source code in the page is allowed to play video, also with audio track, on both players. This works also on mobile, e.g. Safari iOS, which is quite *picky* in terms of what source code can/cannot do automatically without user request.

*On the way issue is described:*

Just to note that what you put in *Reduced test case* and *Steps to reproduce* are really still only part of the description, which is good to have anyway.  
In *Reduced test case* one should provide a piece of code, or a link to codepen, or to a minimal project that actually shows what you did and the problem that arises.  
In *Steps to reproduce* one should provide the steps on how, given that code/codepens/..., to arrive at the point where the issue shows up.  
Please forgive if part of that was already known to you.  
This issue description is an example: #6232 

Hope that helps. Bye
